test/ruleTestRunner.ts

```diff
@@ -25,7 +25,12 @@ console.log();
 console.log(colors.underline("Testing Lint Rules:"));
 /* tslint:enable:no-console */
 
-const testDirectories = glob.sync("test/rules/**/tslint.json").map(path.dirname);
+// Support running a subset of lint rules:
```

Comment:
is this change necessary?

you can run rule tests by simply using `./bin/tslint --test some/glob*pattern`